### PR TITLE
Fix - Update onboarding guide commands

### LINKED ISSUE #213: Bug - Update the onboarding commands to new commands
# Issue
Since #208 was merged in, the default command names for contact operations, eg. `add` has become `addperson`, etc. Change the onboarding steps to reflect this new change.

### PR DESCRIPTION
**Pull request by Wei-Xinn**
## Changes
This PR updates the onboarding guide to match the updated person commands.

## Closes
- closes #213

### CODE REVIEW

File: src/main/java/seedu/contax/ui/onboarding/OnboardingStoryManager.java [hunk 1]
```diff
@@ -71,8 +71,8 @@ private void createStories() {
 
 
         test.addStory(new OnboardingStep(
-                "Follow the format 'add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS'"
-                        + "\n\nExample: add n/Johnny p/91234567 e/Johnny@j.com a/Johnny street"
+                "Follow the format 'addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS'"
```
Comment (dandaandaaaaaan) [inline src/main/java/seedu/contax/ui/onboarding/OnboardingStoryManager.java:74]: the command is `addperson` no capitalisation

**Review by dandaandaaaaaan — CHANGES_REQUESTED** [review]
one slight mistake to amend thanks

File: src/main/java/seedu/contax/ui/onboarding/OnboardingStoryManager.java (hunk 1 above)
Comment (Wei-Xinn) [reply src/main/java/seedu/contax/ui/onboarding/OnboardingStoryManager.java:74]: woops thanks

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm!

**Review by KwanHW — APPROVED** [review]
LGTM

**Pull request merged**